### ISSUE TITLE
Doesn't generate pages for tags

### ISSUE THREAD

**Issue by SeanMcP** [11ty/eleventy-base-blog#28]
On a fresh clone and build, we get a `tags/index.html` with the following:

```html
<h1>Tags</h1>
<a href="/tags/another-tag/" class="tag">another-tag</a>
<a href="/tags/second-tag/" class="tag">second-tag</a>
<a href="/tags/number-2/" class="tag">number-2</a>
```

However, none of those tag pages are generated.

**Comment by zachleat** [comment]
Ack, that's troubling! I'll have a look

**Comment by SeanMcP** [comment]
I see you're on it, but I just wanted to say that everything works as expected after rolling back to 0.7.1.

**Comment by SeanMcP** [comment]
Fixed in v0.8.1: https://github.com/11ty/eleventy/releases/tag/v0.8.1

**Comment by zachleat** [comment]
Thank you for reporting!
